### DISCUSSION TITLE
Unity VR Game Build Hiccup,Unity VR Game Hiccup

### DISCUSSION THREAD

**Discussion by Ifebube** [community/community#56135] [Programming Help]
![Screen Shot 2023-05-20 at 21 54 08](https://github.com/community/community/assets/30638140/da5f11d3-97c6-4399-b49a-20609e2f8f92)
![WhatsApp Image 2023-05-20 at 18 59 19](https://github.com/community/community/assets/30638140/da616ed1-07bb-46ac-a58d-6ffaf6e66fed)
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

So I'm creating a VR game for Google Cardboard utilising the 2022 version. I recently updated my Unity version and installed a package named GUI after downloading the Google Cardboard SDK. I've been attempting to build the game for Android, but it keeps giving me this error message about my AndroidManifest.xml script (I've tried developing for Desktop and it succeeded). Another terrifying thing it displays is a GUI script error from GUIUtility.cs. Every time I try to run this specific script to retrieve the error, a pop-up message stating "File not Found" appears. My Android SDK has been upgraded, and Visual Studio is my IDE. I'm not sure what the issue is.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ifebube ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by nepal143** [comment]
It looks like the problem might be related to the Google Cardboard SDK interfering with your AndroidManifest.xml file, causing a conflict. You can check the Assets/Plugins/Android/ folder to ensure the AndroidManifest.xml doesn’t cause issues with your project.

As for the GUIUtility.cs error, it could be a conflict between the Google Cardboard SDK and the GUI package. Try removing the GUI package or rebuilding the project by re-importing assets.

Since I mostly work with Oculus, I’m not too familiar with Google Cardboard, but these steps should help!
